src/validation.cpp

```diff
@@ -424,11 +424,18 @@ namespace {
 class MemPoolAccept
 {
 public:
-    explicit MemPoolAccept(CTxMemPool& mempool, Chainstate& active_chainstate) : m_pool(mempool), m_view(&m_dummy), m_viewmempool(&active_chainstate.CoinsTip(), m_pool), m_active_chainstate(active_chainstate),
-        m_limit_ancestors(m_pool.m_limits.ancestor_count),
-        m_limit_ancestor_size(m_pool.m_limits.ancestor_size_vbytes),
-        m_limit_descendants(m_pool.m_limits.descendant_count),
-        m_limit_descendant_size(m_pool.m_limits.descendant_size_vbytes) {
+    explicit MemPoolAccept(CTxMemPool& mempool, Chainstate& active_chainstate) :
+        m_pool(mempool),
+        m_view(&m_dummy),
+        m_viewmempool(&active_chainstate.CoinsTip(), m_pool),
+        m_active_chainstate(active_chainstate),
+        m_limits{
+            .ancestor_count = m_pool.m_limits.ancestor_count,
+            .ancestor_size_vbytes = m_pool.m_limits.ancestor_size_vbytes,
+            .descendant_count = m_pool.m_limits.descendant_count,
+            .descendant_size_vbytes = m_pool.m_limits.descendant_size_vbytes
+        }
```

Comment:
nit in 55b8e6f6cb:

Why not just use default copy?
```suggestion
        m_limits{m_pool.m_limits}
```

Reply:
That's definitely how a sane person would do it yes, woops. Thank you - updated!